src/mixins/date-mixin.ts

```diff
@@ -86,29 +164,12 @@ export default class DateMixin extends Vue {
    * @returns NaN in case of error
    */
   daysFromToday (date: string): number {
+    // safety check
     if (!date) return NaN
+
     // calculate difference between start of "today" and start of "date" (in local time)
-    const todayLocalMs = new Date().setHours(0, 0, 0, 0)
+    const todayLocalMs = new Date(this.currentDate).setHours(0, 0, 0, 0)
```

Comment:
Now uses Current Date (as set in App.vue from server time) instead of whatever the user's browser/OS clock is set to.